### DISCUSSION TITLE
专业版蓝灯的本地文件彻底删掉后重新安装就变成免费版了

### DISCUSSION THREAD

**Discussion by blurice** [getlantern/lantern-forum-cn#425] [Q&A] [answered]
请尽快帮我回复一下。

**Comment by blurice** [comment]
我去，没看到还能邮箱登录，已解决。

**Comment by blurice** [comment]
但愿这次能让网络稳定一点。

**Reply by MaxMKMax** [reply, marked as answer]
@blurice 
好的好的. 您也可以下载最新版本来获取更好的体验.

我们这周发布了新的 Flutter 桌面应用程序测试版（8.0.0 版本）。

Mac - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.dmg

Windows - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.exe

Linux - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview-64-bit.deb

**Reply by blurice** [reply]
thanks bro

**Reply by MaxMKMax** [reply]
@blurice You are welcome bro. Thanks for supporting Lantern, we are glad to have you as our user.
